docset/winserver2025-ps/smbshare/Remove-SmbClientCertificateMapping.md

```diff
@@ -0,0 +1,386 @@
+ [<CommonParameters>]
+```
+
+## DESCRIPTION
+
+The `Remove-SmbClientCertificateMapping` cmdlet removes an existing client certificate mapping for
+the SMB protocol. These mappings are used to authenticate clients that connect to SMB servers using
+certificates. You can use this cmdlet to remove a mapping between a client certificate and an SMB
+client name.
+
+## EXAMPLES
+
+### Example 1: Remove all certificate mappings from a specific store
+
+```powershell
+Remove-SmbClientCertificateMapping -StoreName "MyCertificateStore" -Force
+```
+
+This force removes all certificate mappings that have been stored in a specific certificate store.
+
+### Example 2: Remove a certificate mapping by thumbprint
+
+```powershell
+Remove-SmbClientCertificateMapping -Thumbprint "MyThumbprint" -IncludeHidden -Force
+```
+
+This force removes a certificate mapping with a specific thumbprint, including any hidden
+certificate mappings.
+
+## PARAMETERS
+
+### -AsJob
+
+Runs the cmdlet as a background job. Use this parameter to run commands that take a long time to
+complete.
+
+```yaml
+Type: SwitchParameter
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -CimSession
+
+Runs the cmdlet in a remote session or on a remote computer. Enter a computer name or a session
+object, such as the output of a [New-CimSession](https://go.microsoft.com/fwlink/p/?LinkId=227967)
+or [Get-CimSession](/powershell/module/cimcmdlets/get-cimsession) cmdlet. The default is the
+current session on the local computer.
+
+```yaml
+Type: CimSession[]
+Parameter Sets: (All)
+Aliases: Session
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -DisplayName
+
+Specifies a friendly name to display for the mapping.
+
+```yaml
+Type: String[]
+Parameter Sets: Query
+Aliases:
+
+Required: False
+Position: 4
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### -Flags
+
+Specifies if Named Pipes are enabled for SMB over QUIC. The acceptable values for this parameter
+are:
+
+- `None`: Remove all flags.
+- `AllowNamedPipe`: Enable use of named pipes in SMB over QUIC connections for this mapping (off by
+  default, overrides the value of RestrictNamedPipeAccessOverQuic).
+- `DefaultCert`: Not used.
+
+```yaml
+Type: Flags[]
+Parameter Sets: Query
+Aliases:
+Accepted values: None, AllowNamedPipe, DefaultCert
+
+Required: False
+Position: 7
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### -Force
+
+Forces the command to run without asking for user confirmation.
+
+```yaml
+Type: SwitchParameter
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -IncludeHidden
```

Comment:
not sure if this is true, I will get back to you

Reply:
@nedpyle - as discussed

Reply:
This parameter does nothing and we will remove it. For now can say "This parameter is not used" or to that effect